Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM node:14-alpine
 WORKDIR /app
 COPY . .
 
-RUN apk add --no-cache --virtual .build-deps alpine-sdk python2 git openjdk8-jre
+RUN apk add --no-cache --virtual .build-deps alpine-sdk python2 git openjdk8-jre cmake
```

Comment:
Strange, I though we changed this to `python3`?

Reply:
I think we patched directly to `master`. I'll make the change here.